.github/workflows/branch_preparation.yml

```
@@ -27,9 +27,8 @@ jobs:
 
       - name: Modify the chart version based on the tag
         run: |
-          tag=${{ github.ref_name }}
-          echo "BASE=$(nix-shell --pure --run "./scripts/update-chart-version.sh --tag $tag" ./shell.nix | tail -n 1)" >> $GITHUB_ENV
-
+          nix-shell --pure ./shell.nix --run "./scripts/update-chart-version.sh --tag ${{ github.ref_name }} > /tmp/file"
+          echo "BASE=$(cat /tmp/file)" >> $GITHUB_ENV
```

A more specific name just in case?

```suggestion
          nix-shell --pure ./shell.nix --run "./scripts/update-chart-version.sh --tag ${{ github.ref_name }} > /tmp/openebs-file"
          echo "BASE=$(cat /tmp/openebs-file)" >> $GITHUB_ENV
```